docs/source/getting-started/setting-up-virtualization-environment.adoc

```diff
@@ -44,18 +44,36 @@ For more information, see the GitHub documentation of the link:https://github.co
 === On Ubuntu
 
 .  Install *libvirt* and *qemu-kvm* on your system:
+- For Ubuntu-18.10
++
+----
+$ sudo apt install qemu qemu-kvm virt-manager
```

Comment:
Not sure if this is really the right choice. Feels like `virt-manager` might install a bunch of dependencies that made `libvirt` to work as a daemon. I'd rather describe the least needed options... (and allow `virt-manager` as an alternative).

Reply:
The needed package is now called `libvirt-daemon`: https://salsa.debian.org/libvirt-team/virt-manager/blob/debian/sid/debian/control#L21

Reply:
Agree with @gbraad virt-manager's purpose is more than just installing required bits to run a virtual machine. So you should drop it as we should document packages those are required to run a virtual machine on top of KVM.

Reply:
@LalatenduMohanty, so `sudo apt install qemu qemu-kvm libvirt-daemon` should be enough? I can retry on the clean system and modify the PR if it works.

Reply:
you could, but it is also asking a lot to do this. Over the weekend I'll setup a VM to verify this.

Reply:
@gbraad, that's ok, kinda fun for me, I've got clean Xubuntu 18.10 snapshot, so it is not timeconsuming to try it out. I've just tried `sudo apt install qemu-kvm libvirt-daemon`, but it seems not enough, had to install `libvirt-daemon-system` package also to get it work, so `sudo apt install qemu-kvm libvirt-daemon libvirt-daemon-system` did the trick.